dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracedRequestListener.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.tracing
+
+import io.opentracing.Span
+import okhttp3.Request
+
+/**
+ * Provides the ability to intercept and customise
+ * the tag values in the Span built around [Request].
+ */
```

Comment:
Listener for metrics events.
```suggestion
/**
 * Listener for automatically created [Span] around OkHttp [Request].
 */
```